README.md

```diff
@@ -44,47 +44,56 @@ Getting started with the Twilio API couldn't be easier. Create a
 
 ### API Credentials
 
-The Twilio `RestClient` needs your Twilio credentials. You can either pass these directly to the constructor (see the code below), or use environment variables.
+The Twilio `RestClient` needs your Twilio credentials. We recommend storing them as environment variables, so that you don't have to worry about committing and accidentally posting them somewhere public. See http://twil.io/secure for more details on how to store environment variables.
 
 ```go
 package main
 
 import "github.com/twilio/twilio-go"
 
 func main() {
-    accountSid := "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
-    authToken := "YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY"
-    client := twilio.NewRestClientWithParams(twilio.RestClientParams{
-    	Username: accountSid,
-    	Password: authToken,
-    })
+	// This will look for `TWILIO_ACCOUNT_SID` and `TWILIO_AUTH_TOKEN` variables inside the current environment to initialize the constructor
+	// You can find your Account SID and Auth Token at twilio.com/console
+	// `TWILIO_ACCOUNT_SID` should be in format "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    client := twilio.NewRestClient()
 }
 ```
-Alternatively, a `RestClient` constructor without these parameters will look for `TWILIO_ACCOUNT_SID` and `TWILIO_AUTH_TOKEN` variables inside the current environment.
 
-We suggest storing your credentials as environment variables. Why? You'll never have to worry about committing your credentials and accidentally posting them somewhere public.
+If you don't want to use environment variables, you can also pass the credentials directly to the constructor as below. 
+
 ```go
 package main
 
 import "github.com/twilio/twilio-go"
 
 func main() {
-    client := twilio.NewRestClient()
+    accountSid := "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    authToken := "YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY"
+    client := twilio.NewRestClientWithParams(twilio.ClientParams{
+    	Username: accountSid,
+    	Password: authToken,
+    })
 }
 ```
-Make API calls with a SubAccount by setting the `AccountSid` field in the `twilio.RestClientParams`:
+
+### Using SubAccount
```

Comment:
nit: should be `Subaccount` (lowercase 'a')

Reply:
Fixed.